### PR TITLE
Edited readme file

### PR DESCRIPTION
**Pull request by blackplane**
These changes in the readme aim to make the instructions where to run which command to build bindings and apps more clear.

### CODE REVIEW

**Review by vkubiv — APPROVED** [review]

File: cmd/wallet-sdk-gomobile/README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ This package contains the `gomobile`-compatible version of the SDK. It acts as a
 * The gomobile tools:
   ```
   go install golang.org/x/mobile/cmd/gomobile@latest
-  gomobile init
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/README.md:11]: Why did you remove this step?

File: cmd/wallet-sdk-gomobile/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Wallet SDK Gomobile
+# Wallet SDK Gomobile
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/README.md:1]: Extra space at the end?

File: cmd/wallet-sdk-gomobile/README.md [hunk 3]
```diff
@@ -26,6 +25,12 @@ This package contains the `gomobile`-compatible version of the SDK. It acts as a
 
 ## Generating the Bindings
 
+Change to the gomobile directory:
+```
+cd cmd/wallet-sdk-gomobile
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/README.md:30]: This README is already in `cmd/wallet-sdk-gomobile` (not at the root of the project, where we have a different README), so my assumption is that the user would already be in the `cmd/wallet-sdk-gomobile` folder.

**Comment by DRK3** [comment]
Hi @blackplane, we haven't heard back from you for awhile - can we close this PR, or did you want to discuss any changes?

**Comment by rolsonquadras** [comment]
> Hi @blackplane, we haven't heard back from you for awhile - can we close this PR, or did you want to discuss any changes?

Closing the PR as there is no activity.
